package domain.exceptions

class InviteAlreadySentException: Exception("An invite has already been sent to this user")